src/System.Data.Common/src/ApiCompatBaseline.net45.txt

```diff
@@ -0,0 +1,15 @@
+CannotAddAbstractMembers : Member 'System.Data.Common.DbDataReader.Close()' is abstract in the implementation but is missing in the contract.
```

Comment:
I'm seeing most of these are fixed in 4.5.1, not 4.5.  @saurabh500 / @terrajobst was System.Data.Common supposed to support 4.5 or 4.5.1?